index.js

```diff
@@ -805,4 +805,21 @@ Peer.prototype._transformConstraints = function (constraints) {
   return constraints
 }
 
+var errorCodes = {
+  ERR_CREATE_OFFER: 'ERR_CREATE_OFFER',
+  ERR_CREATE_ANSWER: 'ERR_CREATE_ANSWER',
+  ERR_SET_LOCAL_DESCRIPTION: 'ERR_SET_LOCAL_DESCRIPTION',
+  ERR_SET_REMOTE_DESCRIPTION: 'ERR_SET_REMOTE_DESCRIPTION',
+  ERR_ADD_ICE_CANDIDATE: 'ERR_ADD_ICE_CANDIDATE',
+  ERR_ICE_CONNECTION_FAILURE: 'ERR_ICE_CONNECTION_FAILURE',
+  ERR_SIGNALING: 'ERR_SIGNALING',
+  ERR_DATA_CHANNEL: 'ERR_DATA_CHANNEL'
+}
+
```

Comment:
If these are not exported as constants, is this object even needed?
I'd use plain strings instead or export them as `Peer.ERR_*` constants with numeric codes.

Reply:
+1 for plain strings. No need for another layer.

Reply:
Yeah, I deliberated over that a bit.  My thinking was that I prefer having them all defined somewhere together so there is less chance of 'rogue' error codes popping up without being noticed.  Exporting constants on Peer might be a happy medium.

Reply:
Pushed a new commit with string for the error codes.  Is it normal for the build to hang and fail like this?  Tests passed for me locally but they do stop for quite awhile at some points.

Reply:
`electron-webrtc` has some issues, this is why it hangs. Not your fault.